responses/__init__.py

```diff
@@ -598,6 +604,10 @@ def add_passthru(self, prefix):
 
         >>> responses.add_passthru(re.compile('https://example.com/\\w+'))
         """
+        warnings.warn(
+            "This feature is deprecated in favor of using a PassthroughResponse",
```

Comment:
The proposed feature can't replace this method as many applications of `add_passthru` are not defining individual responses. Instead they allow all traffic for a specific host or path to pass through.

There might be a solution where `add_passthru` creates passthrough responses. That would be more backwards compatible and allow the definition of specific mocks that also passthrough traffic.

Reply:
Good point. Here, `add_passthru` could simply create `PassThroughResponse` instances. The only problem for backward compatibility that I see is the method argument of the `BaseResponse` class, because there is no `ANY` method, but the current behavior of `add_passthru` would require it. I could imagine to implement this by using an `enum.IntFlag` so the response methods could be or-ed, e.g. `responses.PUT | responses.PATCH`. Then `ANY` would be the union of all methods. However, I need to investigate first how this works using older versions of python, because `enum.IntFlag` came with python 3.6.